core/blockchain_test.go

```diff
@@ -3848,17 +3849,17 @@ func testEIP1559Transition(t *testing.T, scheme string) {
 	state, _ := chain.State()
 
 	// 3: Ensure that miner received only the tx's tip.
-	actual := state.GetBalance(block.Coinbase())
+	actual := state.GetBalance(block.Coinbase()).ToBig()
 	expected := new(big.Int).Add(
 		new(big.Int).SetUint64(block.GasUsed()*block.Transactions()[0].GasTipCap().Uint64()),
-		ethash.ConstantinopleBlockReward,
```

Comment:
I think this is kind of lazy here. Should we just compute the expected value as uint256 as well and then compare?

Reply:
If we do it in `big.Int`, then the test can simply do the calculations, and not worry about overflows or negatives. You might call it lazy, but on the other hand, it means that the test becomes less encumbered with overflow spot-checks.

All in all, I think it's better the way it is (I started changing it, and didn't find the changed code to be an improvement)